api/v1_claim_rewards.go

```diff
@@ -225,65 +206,169 @@ func getValidatorAttestation(args GetValidatorAttestationParams) (*SenderAttesta
 }
 
 // Gets reward claim attestations from AAO and Validators in parallel.
+// Now handles validator selection internally with retry logic for failures.
```

Comment:
nit: remove "Now"

or really just update the top comment: `...in parallel, handling retries and selection internally`

Reply:
lmao embarrassed at this comment ngl